tests/rptest/services/redpanda.py

```diff
@@ -234,8 +234,13 @@ def __init__(self,
                  cloud_storage_region="panda-region",
                  cloud_storage_bucket=f"panda-bucket-{uuid.uuid1()}",
                  cloud_storage_api_endpoint="minio-s3",
-                 cloud_storage_api_endpoint_port=9000,
-                 cloud_storage_cache_size=160 * 1000000):
+                 cloud_storage_api_endpoint_port=443,
+                 cloud_storage_cache_size=160 * 1000000,
+                 cloud_storage_enable_remote_read=True,
+                 cloud_storage_enable_remote_write=True,
+                 cloud_storage_reconciliation_interval_ms=None,
+                 cloud_storage_max_connections=None,
+                 cloud_storage_disable_tls=False):
```

Comment:
Should the `cloud_storage_` stuff be enabled by default, or should we just inject extra config for SI test cases?

Did you also change the minio bind port so the `_endpoint_port` change here works for both environments?

Reply:
> Should the `cloud_storage_` stuff be enabled by default, or should we just inject extra config for SI test cases?
The idea is to use SISetting when you want to pass SI configs to RP. The configs are then added to the cluster wide config `extra_rp_conf`. 
Also, AFAIK, the only configs that the user needs to give are the 5 required to get SI working:
```
cloud_storage_enabled: true
cloud_storage_access_key: ***
cloud_storage_secret_key: ***
cloud_storage_region: eu-north-1
cloud_storage_bucket: pandabucket
```
So I can mod SISettings to make sure those configs are required and the rest can be optional.
 
> Did you also change the minio bind port so the `_endpoint_port` change here works for both environments?

Yes. I modified it so Minio is the default endpoint unless AWS creds are provided.

Reply:
I did something similar in this PR: https://github.com/redpanda-data/redpanda/pull/1545/files#diff-461a5e720dfe1a39427569dc9b21083837b4a25712a1b22209da87eb8bda8a52R144

The script was intended to run manually but it was able to use AWS or minio depending on provided environment variables. I totally forget about it, sorry. Maybe it'd be helpful few days ago.